src/main/java/thePackmaster/cards/WitchesStrike/ChitteringPunt.java

```diff
@@ -30,37 +28,34 @@ public ChitteringPunt() {
-            if (!mo.isDeadOrEscaped()){
-                Wiz.vfx(new SmallLaserEffect(Wiz.p().hb.cX,Wiz.p().hb.cY,mo.hb.cX,mo.hb.cY));
-                Wiz.atb(new DamageAction(mo, new DamageInfo(AbstractDungeon.player, damage,
-                        DamageInfo.DamageType.NORMAL), AbstractGameAction.AttackEffect.BLUNT_HEAVY));
-                Wiz.atb(new AbstractGameAction() {
-                    @Override
-                    public void update() {
-                        AbstractDungeon.effectList.add(new StarEffect(mo.hb.cX,mo.hb.cY,Color.CYAN,1.2f));
-                        AbstractDungeon.effectList.add(new StarEffect(mo.hb.cX,mo.hb.cY,Color.CYAN,1.2f));
-                        AbstractDungeon.effectList.add(new StarEffect(mo.hb.cX,mo.hb.cY,Color.CYAN,1.2f));
-                        isDone= true;
+        Wiz.atb(new AbstractGameAction() {
+            @Override
+            public void update() {
+                for (AbstractMonster mo : AbstractDungeon.getCurrRoom().monsters.monsters){
+                    if (!mo.isDeadOrEscaped()){
+                        AbstractDungeon.effectsQueue.add(new SmallLaserEffect(Wiz.p().hb.cX,Wiz.p().hb.cY,mo.hb.cX,mo.hb.cY));
                     }
-                });
-                if (upgraded){
-                    Wiz.vfx(new SmallLaserEffect(Wiz.p().hb.cX,Wiz.p().hb.cY,mo.hb.cX,mo.hb.cY));
-                    Wiz.atb(new DamageAction(mo, new DamageInfo(AbstractDungeon.player, damage,
-                            DamageInfo.DamageType.NORMAL), AbstractGameAction.AttackEffect.BLUNT_HEAVY));
-                    Wiz.atb(new AbstractGameAction() {
-                        @Override
-                        public void update() {
-                            AbstractDungeon.effectList.add(new StarEffect(mo.hb.cX,mo.hb.cY,Color.CYAN,1.2f));
-                            AbstractDungeon.effectList.add(new StarEffect(mo.hb.cX,mo.hb.cY,Color.CYAN,1.2f));
-                            AbstractDungeon.effectList.add(new StarEffect(mo.hb.cX,mo.hb.cY,Color.CYAN,1.2f));
-                            isDone= true;
-                        }
-                    });
                 }
+                isDone = true;
             }
+        });
+        Wiz.doAllDmg(this, AbstractGameAction.AttackEffect.BLUNT_LIGHT,false);
+
+        if (upgraded){
+            Wiz.atb(new AbstractGameAction() {
+                @Override
+                public void update() {
+                    for (AbstractMonster mo : AbstractDungeon.getCurrRoom().monsters.monsters){
+                        if (!mo.isDeadOrEscaped()){
+                            AbstractDungeon.effectsQueue.add(new SmallLaserEffect(Wiz.p().hb.cX,Wiz.p().hb.cY,mo.hb.cX,mo.hb.cY));
+                        }
+                    }
+                    isDone = true;
+                }
+            });
+            Wiz.doAllDmg(this, AbstractGameAction.AttackEffect.BLUNT_LIGHT,false);
```

Comment:
Would change that to use the base game system, the Wiz methods take a few shortcuts.
```suggestion
            addToBot(new DamageAllEnemiesAction(p, multiDamage, damageTypeForTurn, AbstractGameAction.AttackEffect.BLUNT_LIGH));
```
Also don't forget to set isMultiDamage = true in the ctor